### PR TITLE
Update area-owners and fabricbot to reflect libraries area pod changes

### PR DESCRIPTION
**Pull request by jeffhandley**
1. In the area-owners.md file, show the GitHub area teams for all of the Libraries areas instead of individuals' names
    - Remove intra-team consultants from the notes; these are also covered by the area teams
    - With a couple of nit-picky spacing changes too
2. Update the fabricbot config to reflect Jeremy joining Carlos's pod in Santi's place
    - Jeremy remains on the pod with Levi as well
3. Remove all of the old project automation from area pods' boards that are no longer in use

### CODE REVIEW

**Comment by jeffhandley** [comment]
Side note, @eiriktsarpalis... It looks like having the fabricbot config generation files inside the `eng` folder results in a full CI run. That encourages me to move those files elsewhere--maybe to `/.github/fabricbot/`?

File: .github/fabricbot.json [hunk 1]
```diff
@@ -8308,7 +5373,7 @@
               {
                 "name": "labelAdded",
                 "parameters": {
-                  "label": "needs more info"
```
Comment (eiriktsarpalis) [inline .github/fabricbot.json]: How did I miss that? 🙀  Thanks

**Review by eiriktsarpalis — APPROVED** [review]

**Comment by eiriktsarpalis** [comment]
> That encourages me to move those files elsewhere--maybe to /.github/fabricbot/

Good idea -- it would also improve discoverability of the generator tools.

File: .github/fabricbot.json (hunk 1 above)
Comment (jeffhandley) [reply .github/fabricbot.json]: Yeah, I was surprised this "needs more info" had slipped through too. 🤷‍♂️ But we've fixed the glitch. :wink:

**Pull request merged**